css/style.css

```diff
@@ -662,6 +677,16 @@ button.button-inline:hover {
 	cursor: pointer;
 	display: block;
 }
+#boardlist .app-popover-menu-utils {
+	width: 30px;
+	display: inline;
+	position: relative;
+}
+
+.popovermenu ul {
```

Comment:
This style already exists in core css that is served alongside, so perhaps it;s not needed?

```
.bubble ul, .app-navigation-entry-menu ul, .popovermenu ul {
    display: flex !important;
    flex-direction: column;
}
```